src/packages/ratmama/ratmama_parser.py

```diff
@@ -167,39 +172,48 @@
         sep = '-'
 
     if not sep:
-        ctx.bot.board.append(Rescue(irc_nickname=ctx.user.nickname, client=ctx.user.nickname))
+        ctx.bot.board.append(Rescue(irc_nickname=ctx.user.nickname.split("[")[0],
+                                    client=ctx.user.nickname))
         index = ctx.bot.board.find_by_name(ctx.user.nickname)
         await ctx.reply(f"Case #{index} created for {ctx.user.nickname}, please set details")
         return
     system: str = None
-    code_red: bool = False
+    cr: bool = False
```

Comment:
[Suggestion] I think `code_red` was a better name for this variable.

Reply:
[Error][scope][style]
this was renamed to `code_red` intentionally.